src/app/connections/exceptions-log/exceptions-log.component.html

```diff
@@ -0,0 +1,66 @@
+<ng-container id="ShowError" *ngIf="pageStatus === 'error'">
+  <div data-test="ErrorMessage" class="mt-20 d-flex justify-content-center">
+    Exceptions data failed to load.
+  </div>
+</ng-container>
+<ng-container id="ShowLoadingSpinner" *ngIf="pageStatus === 'loading'">
+  <div
+    class="d-flex align-content-center justify-content-center align-items-center progress-spinner"
+  >
+    <mat-spinner></mat-spinner>
+  </div>
+</ng-container>
+
+<ng-container id="NoExceptionDataFound" *ngIf="pageStatus === 'empty'">
+  <div
+    data-test="NoExceptionsMessage"
+    class="mt-20 d-flex justify-content-center"
+  >
+    No exceptions found.
+  </div>
+</ng-container>
+
+<ng-container id="ExceptionDataFound" *ngIf="pageStatus === 'ok'">
+  <h2 class="ml-20 mt-20">
+    List of errors from unsuccessful connection updates made today by currennt
```

Comment:
"current"

Reply:
Also just "unsuccessful connections" sounds smoother I think (I know technically these are for disconnections too but . . .)